src/main/java/Task.java

```diff
@@ -0,0 +1,28 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+
+    public void markAsDone(){
+        isDone = true;
+    }
+
+    public void markAsNotDone(){isDone = false;}
+
+    public String getIcon(){
+        return " ";
+    }
+
+    public String getStart(){return "";}
```

Comment:
Even though these are simple 1 liners, please do still follow the proper style (aka Egyptian Style).